haystack/modeling/model/language_model.py

```diff
@@ -1578,11 +1578,11 @@ def disable_hidden_states_output(self):
 
 class DebertaV2(LanguageModel):
     """
-    This is a wrapper around the DebertaV2 model from HuggingFace's transformers library.
+    This is a wrapper around the DebertaV2 model from Hugging Face's transformers library.
     It is also compatible with DebertaV3 as DebertaV3 only changes the pretraining procedure.
 
     NOTE:
-    - DebertaV2 does not output the pooled_output. An additional pooler is initialized.
+    - DebertaV2 does not output the `pooled_output`. An additional pooler is initialized.
```

Comment:
While this change makes sense, it should probably be repeated throughout this file. Most language models in this file have this documentation for the forward method.

Reply:
Done